.changeset/pretty-forks-smash.md

```diff
@@ -0,0 +1,29 @@
+---
+'astro': major
+---
+
+Removes the `handleForms` prop for the `<ClientRouter />` component
+
+In Astro 4.0, the `handleForms` prop of the `<ClientRouter />` component was deprecated, as it was no longer necessary to opt in to handling `submit` events for `form` elements. This functionality has been built-in by default and the property, if still included in your project, silently had no impact on form submission.
```

Comment:
```suggestion
In Astro 4.0, the `handleForms` prop of the `<ClientRouter />` component was deprecated, as it was no longer necessary to opt in to handling `submit` events for `form` elements. This functionality has been built in by default and the property, if still included in your project, silently had no impact on form submission.
```
We might need to change this in the v6 upgrade guide, too, if this was copied from there!